### ISSUE TITLE
`frame-support` is still in the dependency tree

### ISSUE THREAD

**Issue by JoshOrndorff** [Off-Narrative-Labs/Tuxedo#82]
```bash
$ cargo tree -i frame-support

frame-support v4.0.0-dev (https://github.com/paritytech/substrate.git?tag=monthly-2023-06#6ef184e3)
├── frame-system v4.0.0-dev (https://github.com/paritytech/substrate.git?tag=monthly-2023-06#6ef184e3)
│   └── pallet-transaction-payment v4.0.0-dev (https://github.com/paritytech/substrate.git?tag=monthly-2023-06#6ef184e3)
│       ├── node-template v4.0.0-dev (/home/joshy/ProgrammingProjects/tuxedo/node)
│       └── pallet-transaction-payment-rpc-runtime-api v4.0.0-dev (https://github.com/paritytech/substrate.git?tag=monthly-2023-06#6ef184e3)
│           └── pallet-transaction-payment-rpc v4.0.0-dev (https://github.com/paritytech/substrate.git?tag=monthly-2023-06#6ef184e3)
│               └── node-template v4.0.0-dev (/home/joshy/ProgrammingProjects/tuxedo/node)
└── pallet-transaction-payment v4.0.0-dev (https://github.com/paritytech/substrate.git?tag=monthly-2023-06#6ef184e3) (*)
```

It looks like at least one reason this happens is because we have the rpc for pallet transaction payment. We definitely don't want that, so a first step is to rip that out.

**Comment by JoshOrndorff** [comment]
Solved in #83

**Comment by JoshOrndorff** [comment]
We still depend on `frame-metadata` unfortunately, and I've followed up about that in https://github.com/paritytech/substrate/issues/14296
